src/test/ui/union/union-suggest-field.mirunsafeck.stderr

```diff
@@ -2,13 +2,13 @@ error[E0560]: union `U` has no field named `principle`
   --> $DIR/union-suggest-field.rs:13:17
    |
 LL |     let u = U { principle: 0 };
-   |                 ^^^^^^^^^ help: a field with a similar name exists: `principal`
+   |                 ^^^^^^^^^ help: did you mean `principal`? (a similarly named field)
 
 error[E0609]: no field `principial` on type `U`
   --> $DIR/union-suggest-field.rs:17:15
    |
 LL |     let w = u.principial;
-   |               ^^^^^^^^^^ help: a field with a similar name exists: `principal`
```

Comment:
Just a suggestion: To make the old version a bit shorter, we could do `a similarly named field exists` instead.

Reply:
Personally, I would go in *the opposite* direction and make the typo suggestions *always* verbose (so they appear in their own subdiagnostic window) to make them stand out more, with the current wording. I need to simmer on it to see if it is just familiarity (and the fact I likely wrote the original wording ^_^') or an actual "reason" I prefer it.

The proposed output has one problem though, which is that you now *do* have to read the *whole* thing. When I see `^^^ help: ...` I usually skip to the end to see what the suggested code is. The new message has a question (that we've tried to move away from) *and* a parenthesis that explains what's going on. It *feels* to me as adding verbosity *without* a gain. (But again, I need to argue with myself to see if I can come around to seeing it a different way.)

Reply:
FWIW, error messages are nearly always (and IIUC supposed to be) lowercase, so I don't think we should capitalize.